src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1957,7 +1957,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         WRN_Experimental = 42380
 
         ' // AVAILABLE                             42381 - 49998
-        ERRWRN_Last = WRN_Experimental
+        ERRWRN_NextAvailable = 42381
```

Comment:
Add a smallhelper module at the top of the file.
```vb
    Friend Module ERRID_Helper
        Friend Const START_OF_FEATURES As ERRID = ERRID.FEATURE_AutoProperties
        Friend Const LAST_OF_UNAVAILABLE_ERRWRNIDs As ERRID = ERRID.WRN_Experimental
        Friend Const END_OF_ERRWRN_IDs As ERRID = ERRID.ERRWRN_Last
    End Module
```
And use them as the markers.